Sources/SwiftKuery/Migration.swift

```diff
@@ -69,18 +81,38 @@ public class Migration {
         table1 = from
         table2 = to
     }
-    
-    /// Alter the name of the table to the name in the new version of the table.
-    ///
+    // MARK: Table Alteration
+
+    /**
+     Builds an executes an SQL query to alter the name of the old table to the name in the new version of the table.
+     ### Usage Example: ###
+     In this example, a `Migration` instance is initialized.
+     The alterTableName function is called to initiate the SQL query, which changes the name.
+     The "alterHandler" is a closure which handles the response from the database.
+     ```swift
+     let migration = Migration(from: oldTable, to: newTable, using: connection)
+     migration.alterTableName(alterHandler)
+     ```
+     */
     /// - Parameter onCompletion: The function to be called when the execution of the query has completed.
     public func alterTableName(onCompletion: @escaping ((QueryResult) -> ())) {
         let query = "ALTER TABLE " + Utils.packName(table1._name, queryBuilder: connection.queryBuilder) + " RENAME TO " + Utils.packName(table2._name, queryBuilder: connection.queryBuilder)
         renamed = true
         connection.execute(query, onCompletion: onCompletion)
     }
 
-    /// Add a column to the table.
-    ///
+    /**
+     Create and execute an SQL query to add a `Column` to the new table in the `Migration` instance.
+     ### Usage Example: ###
+     In this example, `Migration` and `Column` instances are initialized.
+     The `alterTableAdd` function is called to add the `Column` instance to the "newTable" within the `Migration` instance.
+     The "alterHandler" is a closure which handles the response from the database.
+     ```swift
+     let migration = Migration(from: oldTable, to: newTable, using: connection)
+     let toDo_title = Column("toDo_title", String.self, notNull: true)
+     migration.alterTableAdd(column: toDo_title, onCompletion: alterHandler)
+     ```
+     */
     /// - Parameter column: The column to add. This should be a column in the new version of the table.
```

Comment:
What are we trying to say here.. that the column will be added to the new version of the table?